core/src/main/java/com/sequenceiq/cloudbreak/converter/spi/StackToCloudStackConverter.java

```diff
@@ -395,6 +407,28 @@ private SpiFileSystem buildAdditionalSpiFileSystem(Stack stack) {
         return spiFileSystem;
     }
 
+    public GroupNetwork buildGroupNetwork(com.sequenceiq.cloudbreak.domain.Network stackNetwork, InstanceGroup instanceGroup) {
+        GroupNetwork groupNetwork = null;
+        InstanceGroupNetwork instanceGroupNetwork = instanceGroup.getInstanceGroupNetwork();
+        if (instanceGroupNetwork != null) {
+            Json attributes = instanceGroupNetwork.getAttributes();
+            Map<String, Object> params = attributes == null ? Collections.emptyMap() : attributes.getMap();
+            Set<GroupSubnet> subnets = new HashSet<>();
+            if (params != null) {
+                List<String> subnetIds = (List<String>) params.getOrDefault(NetworkConstants.SUBNET_IDS, new ArrayList<>());
+                if (subnetIds != null) {
+                    for (String subnetId : subnetIds) {
+                        GroupSubnet groupSubnet = new GroupSubnet(subnetId);
+                        subnets.add(groupSubnet);
+                    }
+                }
+            }
+
+            groupNetwork = new GroupNetwork(stackNetwork.getOutboundInternetTraffic(), subnets, params);
```

Comment:
Not a big deal but if `subnets` is not empty, the subnet IDs will be also contained in `params` as the dynamic attribute `SUBNET_IDS`. Perhaps `SUBNET_IDS` should be removed from the map passed as the third constructor argument to avoid confusions.